src/AppBundle/Form/LoginType.php

```diff
@@ -53,6 +53,6 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getBlockPrefix()
     {
-        return 'app_login';
+        return 'app_admin_login';
```

Comment:
If you need to customize the form name (html attribute and form/data tree key), to ease theming, you can use `block_name` option from the controller, see http://symfony.com/doc/current/reference/forms/types/form.html#block-name.